### ISSUE TITLE
Live Bus Locations Layer

### ISSUE THREAD

**Issue by Denis306** [stadtnavi/stadtnavi_app#153]
The information on how it was implemented in the WebApp is here: https://github.com/stadtnavi/digitransit-ui/issues/369

**Comment by Denis306** [comment]
@SamuelRioTz : The Live Bus Location Layer is going in the public transit layer. 
Once this is done you can see where and how it has been implemented in the map layer switcher.

**Comment by ahmed-nawar** [comment]
I couldn't find any bus positions icons. How can I test this? v1.2.9

**Comment by Denis306** [comment]
@ahmed-nawar We will have to test this tomorrow I think. I am not 100% sure if the buses that send their location are active on the weekends.

**Comment by Denis306** [comment]
I just saw that they do. So it is still not properly working @SamuelRioTz

**Comment by Denis306** [comment]
@ahmed-nawar so apparently there were some problems in the backend or it was not loading. Now it is working in the web and mobile app again.
![image](https://user-images.githubusercontent.com/63004289/122679413-0a952b80-d1eb-11eb-954b-b30257c7deaf.png)

**Comment by ahmed-nawar** [comment]
I can see that the bus locations are working on the IOS app but I can't find them on the dev or beta webapps
@Denis306

**Comment by ahmed-nawar** [comment]
I can see the bus location icons on the webapp now. They were not shown earlier.

**Comment by ahmed-nawar** [comment]
If the webapp is launched while the toggle for **Bus Positions** is **off**, when I turn the toggle on the **Bus Positions** will not show until I refresh the browser.
To reproduce:
- Toggle the checkbox off
- Refresh the browser
- Try to toggle the checkbox on
- It won't show until you refresh the browser

@Denis306 Should I create a separate ticket for this issue?

**Comment by Denis306** [comment]
@ahmed-nawar please do so but in the Frontedn Project. This is a web app issue not a mobile app issue.
